t/Makefile.am

```diff
@@ -54,6 +54,35 @@ TESTS = Test01-check_programs_exist.sh \
         Test22-reference_combine-ncc.sh \
         Test23-reference_decompress-ncc.sh \
         Test24-reference_fregrid.sh \
+        Test32-fregrid_stretched.sh \
+        Test33-fregrid_stretched.sh \
+        Test34-fregrid_stretched.sh \
+        Test35-fregrid_no_stretched.sh \
+        Test36-fregrid_no_stretched.sh \
+        Test37-fregrid_no_stretched.sh \
+	01-make_hgrid.sh \
+        01-make_vgrid.sh
+
+EXTRA_DIST = $(TESTS) \
+             Test02-input\
+             Test02-reference \
+             Test03-input \
+             Test06-input \
+             Test07-input \
+             Test08-input \
+             Test10-input \
+             Test12-input \
+             Test15-input \
+             Test16-input \
+             Test17-input \
+             Test18-input \
+             Test20-input \
+             Test26-input \
+             Test32-input \
+             Test17-reference \
+             Test18-reference \
+             Test20-reference 
+
```

Comment:
Thanks a lot Chris for your questions. Sorry for the late reply.
I think we can for sure combine the 6 into a 1 or 2 in order to separate the stretched from the non-stretched.

There are different lats when running the make_hgrid 30.0, 34.0 and 35.4 I think to stretched with 2.5. Do you mean to add a target_grid  lat/lon to stretched conversion again?.

I'm not sure if we replace the fv_tracer.res.tileX.nc files with a random number, what variable we will use in the --scalar-field?. In this case we using o3. Do you mean --input_file 1 or --input_file 2?.

Reply:
Hi Luis, I wasn't sure if my suggestion makes sense- but it was to exercise the reverse direction within fregrid. So instead of regridding from stretched/not-stretched to lat/lon, we regrid from lat/lon to stretched/not-stretched. If it doesn't make sense let's skip it.

For the random number or 1's in the input files, I think you could name the variable whatever as long as it was consistent with the code to generate it. Sorry, I'm not sure what you mean by "--input_file 1 or input_file 2". The --input_file option specifies the input file to be regridded, right, so it would be whatever name the random number or 1's file was called?